internal/server/router.go

```diff
@@ -16,20 +16,24 @@ func NewHandler(config *HandlerConfig) http.Handler {
 	apiV1Router.HandleFunc("GET /certificate_requests/{id}", adminOrUser(config.JWTSecret, GetCertificateRequest(config)))
 	apiV1Router.HandleFunc("DELETE /certificate_requests/{id}", adminOrUser(config.JWTSecret, DeleteCertificateRequest(config)))
 	apiV1Router.HandleFunc("POST /certificate_requests/{id}/reject", adminOrUser(config.JWTSecret, RejectCertificateRequest(config)))
+	apiV1Router.HandleFunc("POST /certificate_requests/{id}/sign", adminOrUser(config.JWTSecret, SignCertificateRequest(config)))
 	apiV1Router.HandleFunc("POST /certificate_requests/{id}/certificate", adminOrUser(config.JWTSecret, CreateCertificate(config)))
 	apiV1Router.HandleFunc("DELETE /certificate_requests/{id}/certificate", adminOrUser(config.JWTSecret, DeleteCertificate(config)))
-	apiV1Router.HandleFunc("GET /accounts", adminOnly(config.JWTSecret, ListAccounts(config)))
-	apiV1Router.HandleFunc("POST /accounts", adminOrFirstUser(config.JWTSecret, config.DB, CreateAccount(config)))
-	apiV1Router.HandleFunc("GET /accounts/{id}", adminOrMe(config.JWTSecret, GetAccount(config)))
-	apiV1Router.HandleFunc("DELETE /accounts/{id}", adminOnly(config.JWTSecret, DeleteAccount(config)))
-	apiV1Router.HandleFunc("POST /accounts/{id}/change_password", adminOrMe(config.JWTSecret, ChangeAccountPassword(config)))
+
 	apiV1Router.HandleFunc("GET /certificate_authorities", adminOnly(config.JWTSecret, ListCertificateAuthorities(config)))
 	apiV1Router.HandleFunc("POST /certificate_authorities", adminOnly(config.JWTSecret, CreateCertificateAuthority(config)))
 	apiV1Router.HandleFunc("GET /certificate_authorities/{id}", adminOnly(config.JWTSecret, GetCertificateAuthority(config)))
 	apiV1Router.HandleFunc("PUT /certificate_authorities/{id}", adminOnly(config.JWTSecret, UpdateCertificateAuthority(config)))
 	apiV1Router.HandleFunc("DELETE /certificate_authorities/{id}", adminOnly(config.JWTSecret, DeleteCertificateAuthority(config)))
+	apiV1Router.HandleFunc("POST /certificate_authorities/{id}/sign", adminOnly(config.JWTSecret, SignCertificateAuthority(config)))
```

Comment:
I'm unsure about this new endpoint, a Certificate Authority could have multiple certificates over its life right? If that's the case, what does `/sign` sign? I think `/sign` makes sense against a certificate request object, not necessarily against a certificate authority object.